code/modules/mob/living/living_defense.dm

```diff
@@ -340,9 +340,8 @@
 		return
 	if(stat != DEAD && !is_servant_of_ratvar(src))
 		to_chat(src, "<span class='userdanger'>A blinding light boils you alive! <i>Run!</i></span>")
-		adjustFireLoss(35)
 		if(src)

```

Comment:
This if is stupid

Reply:
Some mobs delete on death, so hitting them with fireloss might kill them, wouldn't it?